src/modules/pokemon.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable no-await-in-loop */
+/* eslint-disable object-curly-newline */
+/* eslint-disable no-plusplus */
+/* eslint-disable import/prefer-default-export */
+import '../style.css';
+import like from '../assets/like.png';
+import { addLike, catchLike } from './likes.js';
+
+const $list = document.querySelector('.pokemon-list');
+const url = 'https://pokeapi.co/api/v2/pokemon/';
+const array = [];
+
+const fetchPokemon = async (id) => {
+  await fetch(url + id)
+    .then((response) => response.json())
+    .then((data) => {
+      // eslint-disable-next-line no-use-before-define
+      array.push(data);
+    });
+};
+
+export const displayList = (array) => {
+  array.forEach((poke) => {
+    const { name, id, sprites, types } = poke;
+    $list.innerHTML += `
+        <div class="card">
+            <div class="container-sprite"><img src="${sprites.front_default}" alt="${name}"></div>
+            <p class="name">${name}</p>
+            <p class="types">${types.map((type) => type.type.name).join(', ')}</p>
+            <p class="id">#${id}</p>
+            <div class="buttons-container">
+                <button class="button-comments">Comments</button>
+                <button class="button-like" name="${name}"><p name="${name}" class="counter-likes">0</p><img class="like-img" src="${like}" alt="like">Like!</button>
+            </div>
+        </div>
+      `;
+  });
```

Comment:
Good job the items fetched from api and displayed correctly 👍🏼 👍🏼 , but there are some styling adjustments to do : 

- [ ] the images are stretched to fix this use can apply the “object-fit” CSS property as “cover”.
- [ ] the like, comment and title items are not displayed the same way as in the given layout.
- [ ] it is better to reduce cards size as they doesn't contain a lot of element .

![Capture d’écran du 2023-08-11 19-16-51](https://github.com/SaveryIV/Javascript-Capstone-Project/assets/59707859/40c61d23-f47f-4689-b170-f4cd1e66e85a)

![Capture d’écran du 2023-08-11 19-12-35](https://github.com/SaveryIV/Javascript-Capstone-Project/assets/59707859/082df63c-7589-4d84-bbae-c34c053e8349)